_data/usa_identifier.yml

```diff
@@ -13,8 +13,8 @@ org_primary_url: https://www.gsa.gov/tts/
 org_primary_email: tts-info@gsa.gov
 org_primary_about: https://www.gsa.gov/tts/
 org_primary_bio: "As part of GSA’s Technology Transformation Services (TTS), we apply modern methodologies and technologies to improve the public’s experience with government. We help agencies make their services more accessible, efficient, and effective with modern applications, platforms, processes, personnel, and software solutions."
-org_secondary: Technology Transformation Services
-org_secondary_acronym: TTS
+org_secondary: 18F
```

Comment:
`org_secondary` currently contains information from each of `TTS`, `18F`, and `TTS Solutions` (see `org_secondary_bio` on line 22). What does `org_secondary` mean here? And should we change all of it to reflect `18F` for this PR?

Reply:
I think (and this is my personal opinion as a guide user) primary should be a reference to TTS and secondary should refer to 18F consistently. Looks like `org_primary_bio` and `org_secondary_bio` aren't being used anyway in the templates and can be removed. But as the product owner, it's your call :)